### ISSUE TITLE
Install fails with flask error on Ubuntu 16.04

### ISSUE THREAD

**Issue by mtlynch** [ReinProject/python-rein#116]
When I run 

```text
$ sudo python setup.py install
```

It eventually fails with:

```text
error: The 'flask' distribution was not found and is required by rein, flask-wtf
```

Full output: https://gist.github.com/mtlynch/1b518464933a3986c05a484140df16bc

Tested on Ubuntu 16.04.

**Comment by mtlynch** [comment]
I worked around this with

```shell
$ sudo pip install flask
```

And then re-ran the original install command and it succeeded.

**Comment by weex** [comment]
Thanks. Fixed with bce4df96ce6304a32ec0cf4461674cd573f1bd96
